Don't install external libraries in Dorado's lib directory.
Simon Branford (University of Birmingham)
updated John Dey (Fred Hutch Cancer Center)
diff -ruN dorado.orig/cmake/InstallRedistLibs.cmake dorado/cmake/InstallRedistLibs.cmake
--- dorado.orig/cmake/InstallRedistLibs.cmake	2025-07-16 02:14:42.000000000 +0000
+++ dorado/cmake/InstallRedistLibs.cmake	2025-07-23 20:13:22.409472000 +0000
@@ -54,7 +54,7 @@
         else()
             # bundle the libraries from the cuda toolkit
             file(GLOB NATIVE_CUDA_LIBS "${CUDAToolkit_TARGET_DIR}/targets/${CMAKE_SYSTEM_PROCESSOR}-linux/lib/${LIB}")
-            install(FILES ${NATIVE_CUDA_LIBS} DESTINATION lib COMPONENT redist_libs)
+            #install(FILES ${NATIVE_CUDA_LIBS} DESTINATION lib COMPONENT redist_libs)
         endif()
     endforeach()
 
